greenplumpython/order.py

```diff
@@ -12,7 +10,8 @@
 
 class DataFrameOrdering:
     """
-    Represents an ordered GreenplumPython :class:`~dataframe.DataFrame` generated by
+    Representation of an ordered GreenplumPython :class:`~dataframe.DataFrame` generated by.
```

Comment:
Line 13 and line 15 was ONE sentence before, they cannot be splited.
Change to:

```
Represent an ordered GreenplumPython :class:`~dataframe.DataFrame`.

The order is decided by:
```